lib/utils.pm

```diff
@@ -2066,7 +2066,10 @@ sub cleanup_disk_space {
 
     record_soft_failure "bsc#1192331", "Low diskspace on Filesystem root";
 
-    my @snap_lists = split /\n/, script_output("snapper list --disable-used-space | grep important= | grep -v single | awk \'{print \$1}\'");
+    my $ret = script_run("snapper --help | grep disable-used-space");
+    my $disable = '';
```

Comment:
my $disable = $ret ? '' : '--disable-used-space';

And remove Line 2071

Reply:
Changed and thanks a lot